tests/test_reader.py

```diff
@@ -56,18 +57,22 @@ def query_table_pq(tmp_path):
 
 @pytest.fixture()
 def matrix_class():
-    return dist_reader("genomic_address_service/example/mcluster/hamming/results.text", n_records=10, min_dist=1)
+    return dist_reader("genomic_address_service/example/mcluster/hamming/results.text", n_records=10)
 
```

Comment:
Why `min_dist=1` was removed just for the record?

Reply:
The functionality of `min_dists` was removed in PR #25. There isn't even a way to specify minimum or maximum distances on the command line anymore, so there's no point having it part of the constructor if it doesn't do anything.